Library/Homebrew/rubocops/formula_desc_cop.rb

```diff
@@ -0,0 +1,76 @@
+require_relative "./mixins/on_formula_class"
+
+module RuboCop
+  module Cop
+    module Homebrew
+      # This cop audits `desc` in Formulae
+      #
+      # - Checks for existence of `desc`
+      # - Checks if size of `desc` > 80
+      # - Checks if `desc` begins with an article
+      # - Checks for correct usage of `command-line` in `desc`
+      # - Checks if `desc` contains the formula name
+
+      class FormulaDesc < Cop
+        include FormulaDef
+
+        def on_formula(node, class_node, _parent_class_node, body)
+          check(node, body, class_node.const_name)
+        end
+
+        private
+
+        def check(node, body, formula_name)
+          body.each_child_node(:send) do |call_node|
+            _receiver, call_name, args = *call_node
+            next unless call_name == :desc && !args.children[0].empty?
+            description = args.children[0]
+
+            source_buffer = call_node.source_range.source_buffer
+            line_number = call_node.loc.line
+            line_begin_pos = call_node.source_range.source_buffer.line_range(call_node.loc.line).begin_pos
+            desc_begin_pos = call_node.children[2].source_range.begin_pos
+
+            linelength = "#{formula_name}: #{description}".length
+            if linelength > 80
+              column = desc_begin_pos - line_begin_pos
+              length = call_node.children[2].source_range.size
+              sourcerange = source_range(source_buffer, line_number, column, length)
+              message = <<-EOS.strip_indent
+                        Description is too long. \"name: desc\" should be less than 80 characters.
```

Comment:
Indent this two characters after `message` and use `EOS.undent`. Are you sure you need to escape `\"`?